spec/features/numismatic_reference_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe "NumismaticReference", js: true do
+  let(:user) { FactoryBot.create(:admin) }
+  let(:adapter) { Valkyrie::MetadataAdapter.find(:indexing_persister) }
+  let(:reference) { FactoryBot.build(:numismatic_reference) }
+  let(:change_set_persister) { ChangeSetPersister.new(metadata_adapter: adapter, storage_adapter: Valkyrie.config.storage_adapter) }
+
+  before do
+    sign_in user
+    change_set = NumismaticReferenceChangeSet.new(reference)
+    change_set_persister.save(change_set: change_set)
+  end
+
+  describe "pagination z index" do
+    it "has value 1" do
```

Comment:
Can you change the test descriptions to reflect what the user sees? Something like `it "is not displayed on top of the add content menu"`

Reply:
I did, thank you :)